network-api/networkapi/donate/pagemodels/landing_page.py

```diff
@@ -13,7 +13,7 @@ class DonateLandingPage(BaseDonationPage):
     # Only allow creating landing pages at the root level
     parent_page_types: list = ["wagtailpages.Homepage"]
 
-    subpage_types: list = ["DonateHelpPage"]
+    subpage_types: list = ["DonateHelpPage", "wagtailpages.OpportunityPage"]
```

Comment:
Adding OpportunityPage( AKA "DefaultPage", the page used to create /ways-to-give/), to DonateLandingPage's child page list. 

That way we will be able to create /donate/ways-to-give.